ophyd/userapi/scan_api.py

```
@@ -667,26 +567,22 @@ def post_scan(self):
         lmsg.append('Scan ID         : {}'.format(self.scan_id))
         lmsg.append('')
         lmsg.append(msg)
-        lmsg.append('Triggers:')
-        for trig in self.triggers:
-            lmsg.append('{:<30}'.format(trig.name))
         lmsg.append('')
         lmsg.append('{0:=^80}'.format(''))
-        for p in self.triggers + self.detectors:
+        for p in self.detectors:
             try:
-                lmsg.append('PV:{}'.format(p.report['pv']))
-            except KeyError:
+                [lmsg.append(val) for key, val in p.source().iteritems()]
+            except:
```

Catch-all exceptions are dangerous. Can we be more specific?